src/roktManager.ts

```diff
@@ -85,8 +92,23 @@ export default class RoktManager {
             return Promise.resolve({} as IRoktSelection);
         }
 
+        debugger;
+
         try {
-            return this.kit.selectPlacements(options);
+            let enrichedOptions = options;
+            const { attributes } = options;
+            const sandbox = attributes?.sandbox ?? this.sandbox;
```

Comment:
`options` is typed as `IRoktManagerOptions`, which doesn't have an attributes key.  I think you just need to pull sandbox directly from options, as opposed to looking for it in attributes